### PR TITLE
Add GAS_PRICE_ADDITION

### PR DESCRIPTION
**Pull request by pereztechseattle**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -22,4 +23,4 @@ ETHEREUM_NODE_URL=https://parity.expotrading.com
 NETWORK_ID=1
 ACCOUNT_POLL_INTERVAL_MS=5000
 MARKET_POLL_INTERVAL_MS=5000
-SOLO_LIQUIDATE_POLL_INTERVAL_MS=5000
+SOLO_LIQUIDATE_POLL_INTERVAL_MS=5000
```
Comment (BrendanChou) [inline .env]: nit: removed EOF line

File: .env [hunk 2]
```diff
@@ -5,6 +5,7 @@ GAS_PRICE_MULTIPLIER=2
 GAS_PRICE_UPDATE_FREQUENCY_SEC=120
 INITIAL_GAS_PRICE=30000000000
 GAS_REQUEST_TIMEOUT_MS=10000
+GAS_PRICE_ADDITION=0
```
Comment (BrendanChou) [inline .env]: nit: default to 1

File: .env (hunk 1 above)
Comment (pereztechseattle) [reply .env]: oops, thx.

File: src/lib/gas-price.js [hunk 3]
```diff
@@ -34,7 +34,12 @@ export async function updateGasPrice() {
     throw new Error('GAS_PRICE_MULTIPLIER not specified');
   }
 
-  const totalWei = `${Math.round(Number(fast) * multiplier * 100000000)}`;
+  const addition = Number(process.env.GAS_PRICE_ADDITION || 0);
```
Comment (pereztechseattle) [inline src/lib/gas-price.js]: should this default to 1 also?

**Pull request merged**